mod/metrics/set/abstract/metric_thumbnail.rb

```diff
@@ -14,23 +15,24 @@ def thumbnail_image
   end
 
   def thumbnail_title
-    content = wrap_with(:div, nest(card.metric_title_card,
-                                   view: :name),
-                        class: "ellipsis")
-    return content unless voo.show?(:thumbnail_link)
+    return thumbnail_title_text unless voo.show?(:thumbnail_link)
```

Comment:
fwiw, I'm increasingly favoring the pattern where the top level method has something like:
```
voo.show?(:thumbnail_link) ? thumbnail_title_link : thumbnail_title_text
```
a lot of times some of the most important logic is hidden at the end of a return statement :)

Reply:
not necessarily asking for that change here and now.